README.md

```diff
@@ -305,7 +308,19 @@ This will list a brief report of results in the following format:
 Coded 24 verbal autopsies (out of 30) [6 issues]
 ```
 
-You will receive an error message if pyCrossVA or InterVA5 are unavailable.
+If you'd like to re-code all existing VAs in the system (overwriting previous COD assignments) you can do so by passing `overwrite True` to the command-line call. This will save a backup table of old COD assignments, clear all CODs in the database, and re-run the coding algorithm on all eligible VAs.
+
```

Comment:
By default, old COD assignments will be saved to a .csv file named "old_cod_mapping.csv." You can customize the name of this file by passing `--cod_fname <filename>` as an optional parameter.